README.md

```diff
@@ -64,3 +64,13 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Website](http://thlorenz.com/)
 - [Patreon](https://www.patreon.com/thlorenz)
 
+### Amorelandra
```

Comment:
Missing entry in Table of Contents ?

Reply:
Yes, for the first few I didn't add ToC entries - manually added them later :)